src/bin/zmservicectl

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2010, 2011, 2013, 2014, 2015, 2016 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+# 
+
+if [ x`whoami` != xzimbra ]; then
+	echo Error: must be run as zimbra user
+	exit 1
+fi
+
+ZMJAVA="/opt/zimbra/common/lib/jvm/java/bin/java"
+LD_LIBRARY_PATH="/opt/zimbra/license/lib"
+LICENSE_DAEMON_SERVICE_OUTPUT="/opt/zimbra/log/license-daemon-service.log"
+LICENSE_DAEMON_SERVICE="/opt/zimbra/license/lib/app.jar"
+LICENSE_DAEMON_SERVICE_PORT="8081"
+licenseDaemonServicepid=""
+
+checkLicenseDaemonServiceRunning()
+{
+	licenseDaemonServicepid=`ps -aef | grep ${LICENSE_DAEMON_SERVICE} | grep -v grep |awk '{print $2}'`
+	if [ "x$licenseDaemonServicepid" = "x" ]; then
+		licenseDaemonServiceRunning=0
+	else
+		licenseDaemonServiceRunning=1
+	fi
+} 
+
+startLicenseDaemonService()
+{
+	err=0
+	checkLicenseDaemonServiceRunning
+	echo -n "Starting license daemon service..."
+	if [ $licenseDaemonServiceRunning = 1 ]; then
+		echo "license daemon service is already running."
+		return
+	fi
+	if [ "x${licenseDaemonServicepid}" != "x" ]; then
```

Comment:
if service is already running then it will not come to this point, so when exactly we need to kill process?

Reply:
right, this is not required